### PR TITLE
Fix deprecations introduced in geo-types v0.7.8.

### PR DESCRIPTION
**Pull request by metasim**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/gdal/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ libc = "0.2"
 geo-types = { version = "0.7" }
 gdal-sys = { path = "gdal-sys", version = "^0.8" }
 ndarray = { version = "0.15", optional = true }
-chrono = { version = "0.4" }
+chrono = { version = ">= 0.4.0, <=0.4.22" }
```
Comment (metasim) [inline Cargo.toml:24]: Addressed in #336

**Review by lnicola — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -21,7 +21,7 @@ libc = "0.2"
 geo-types = { version = "0.7" }
```
Comment (lnicola) [inline Cargo.toml:21]:
```suggestion
geo-types = { version = "0.7.8" }
```

File: Cargo.toml (hunk 1 above)
Comment (lnicola) [inline Cargo.toml:24]: Nit, feel free to ignore if you'll change it back in #336:

```suggestion
chrono = { version = ">= 0.4.0, <= 0.4.22" }
```

**Pull request merged**